src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 800px;
+  padding: 10px;
+
+  @extend %start-day-sun;
```

Comment:
The placeholder `%start-day-sun` is being extended here. Ensure that this placeholder is defined elsewhere in your SCSS files. If it's not defined, this will cause a compilation error.